boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -69,30 +69,32 @@ extern "C" {
 /** @} */
 
 /**
- * @name UART configuration
+ * @name    UART configuration
  * @{
  */
-#define UART_NUMOF          (1U)
-#define UART_0_EN           1
-#define UART_1_EN           0
-#define UART_IRQ_PRIO       1
-#define UART_CLK            ROM_SysCtlClockGet()  /* UART clock runs with 40MHz */
-/* UART 0 device configuration */
-#define UART_0_DEV          UART0_BASE
-#define UART_0_CLK          (40000000)
-#define UART_0_IRQ_CHAN     UART0_IRQn
+static const uart_conf_t uart_config[] = {
+    {
+        .dev                = 0,
+        .uart_txint_mode    = UART_TXINT_MODE_EOT,
+        .uart_fifo_tx       = UART_FIFO_TX4_8,
+        .uart_fifo_rx       = UART_FIFO_RX4_8,
+        .uart_irq_chan      = UART0_IRQn,
+        .gpio_sysctl        = SYSCTL_PERIPH_GPIOA,
+        .gpio_port          = GPIO_PORTA_BASE,
+        .pins = {
+            .rx             = GPIO_PA0_U0RX,
+            .tx             = GPIO_PA1_U0TX,
+            .mask_rx        = (GPIO_PIN_0),
+            .mask_tx        = (GPIO_PIN_1),
+        }
+    },
+};
+
+/* interrupt function name mapping */
 #define UART_0_ISR          isr_uart0
-/* UART 0 pin configuration */
-#define UART_0_PORT         GPIOA
-#define UART_0_TX_PIN       UART_PA1_U0TX
-#define UART_0_RX_PIN       UART_PA0_U0RX
-
-/* UART 1 device configuration */
-#define UART_1_DEV          UART1_BASE
-#define UART_1_CLK          (40000000)
-#define UART_1_IRQ_CHAN     UART1_IRQn
-#define UART_1_ISR          isr_uart1
```

Comment:
IMHO you still need to have the defines for UART_X_ISR

Reply:
I would a bit disagree since that is how all other boards seem to only have ISR macros for the UARTs that are supported.  I believe this CPU supports 8 UARTs, 3 have interrupt vectors.  I could port in everything but I think that would be a bit clunky, whatever makes you happy!